ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/HardFork/Combinator.hs

```diff
@@ -138,7 +139,8 @@ prop_simple_hfc_convergence testSetup@TestSetup{..} =
     counterexample ("eraSizeA: " <> show eraSizeA) $
     tabulate "epochs in era A" [labelEraSizeA] $
     prop_general args testOutput .&&.
-    prop_allExpectedBlocks
+    prop_allExpectedBlocks .&&.
+    prop_finalProtVers
```

Comment:
Just to confirm: we don't add an expected failure here because even if this commit won't be squashed, when bisecting we don't care about compiling the tests? Or do we? :thinking: